### PR TITLE
[mypy] fix compression folder

### PR DESCRIPTION
**Pull request by algobytewise**
### **Describe your change:**
Related Issue: #4052

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by cclauss** [comment]
@iko1133 Can you please review the change to `compression/lempel_ziv.py` ?

**Comment by algobytewise** [comment]
I fixed the doctest in "2_hidden_layers_neural_network.py". I think the problem was that it has random initial weights, which is why the doctest sometimes failed.

File: neural_network/2_hidden_layers_neural_network.py [hunk 1]
```diff
@@ -260,8 +260,9 @@ def example() -> int:
     In this example the output is divided into 2 classes i.e. binary classification,
     the two classes are represented by '0' and '1'.
 
-    >>> example()
-    1
+    >>> result = example()
+    >>> result == 0 or result == 1
```
Comment (cclauss) [inline neural_network/2_hidden_layers_neural_network.py:265]:
```suggestion
    >>> result in (0, 1)
```

Comment (cclauss) [inline neural_network/2_hidden_layers_neural_network.py:265]:
```suggestion
    >>> example() in (0, 1)
```

**Review by iko1133 — APPROVED** [review]

**Review by cclauss — APPROVED** [review]

**Pull request merged**